### PR TITLE
FIX Use correct argument for FormBuilder context

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-admin/issues/1778

### CODE REVIEW

File: client/src/components/SearchableDropdownField/SearchableDropdownField.js [hunk 1]
```diff
@@ -92,11 +92,9 @@ const SearchableDropdownField = ({
...
       setJustChanged(true);
     }
-    if (multi) {
-      onChange(val);
-    } else {
-      onChange(val, { value: val.value });
-    }
+    // The first param is for handleChange() SearchableDropdownFieldEntwine.js
+    // The second param is for onChange in FormBuilder::buildComponent()
+    onChange(val, { value: val });
```
Comment (ChloeSartorelli) [inline client/src/components/SearchableDropdownField/SearchableDropdownField.js]: This change has resulted in a regression of https://github.com/silverstripe/silverstripe-admin/issues/1733
Probably still needs to be `val.value`, at least for single fields. Please read the comments in the issue and the PR 'cause I recall there was discussion about the difference between single and multi fields, as well.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

**Comment by GuySartorelli** [comment]
I can't de-select a record from the single `SearchableDropdownField` in an elemental block anymore.
I can select a _new_ record, but not deselect one that was already saved into the field.

**Comment by emteknetnz** [comment]
I think this might just a be bad default when lazy-loading

Things are "clearable" by default when not lazy-loading because there's a default blank option that's selectable. This option is not there however when you lazy-load. I can solve this by setting `->setIsClearable(true)` alongside `->setIsLazyLoaded(true)` then I can clear the value.  

I think the solution here is that we just update the field to be clearable by default? This won't be a breaking change, just a behavioural change we could ship in 5.3

Note that clicking the little 'x' when it's clearable will clear ALL selected options for the mutli version of the field, though this seems like it's still a good default to me

I'll make a new issue for this

**Comment by emteknetnz** [comment]
New issue to ensure field is always clearable https://github.com/silverstripe/silverstripe-admin/issues/1791

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**